cardano-cli/src/Cardano/CLI/Shelley/Parsers.hs

```diff
@@ -1688,6 +1691,9 @@ pTxFile fdir =
         Input -> "tx-file"
         Output -> "out-file"
 
+pAnyTxFile :: Parser (Either TxBodyFile TxFile)
```

Comment:
As @erikd please don't use `Either` like this. We take the `Left` of an either to be an error. Create a new sum type if you must.

Reply:
done